packages/roosterjs-content-model/lib/publicApi/utils/formatSegmentWithContentModel.ts

```diff
@@ -33,19 +40,7 @@ export function formatSegmentWithContentModel(
 
             formats.forEach(format => toggleStyleCallback(format, !isTurningOff));
 
-            const isCollapsedSelection =
-                segments.length == 1 && segments[0].segmentType == 'SelectionMarker';
-
-            if (!pendingFormat && isCollapsedSelection) {
```

Comment:
Actually, after selectWord(), selection is still possible collapsed. That can happen when cursor is on an empty line, ore even the whole doc is empty. So we need to keep this part.

Reply:
The selection is still collapsed but what changes? Do we support adding pending states to empty lines?